src/core/dev/ring_simple.cpp

```diff
@@ -750,11 +748,7 @@ bool ring_simple::is_available_qp_wr(bool b_block, unsigned credits)
                 if (p_cq_mgr_tx) {
 
                     // Allow additional CQ arming now
-                    if (safe_mce_sys().doca_tx) {
-                        m_hqtx->clear_notification();
-                    } else {
-                        p_cq_mgr_tx->reset_notification_armed();
-                    }
+                    p_cq_mgr_tx->reset_notification_armed();
```

Comment:
see in line 717:
"
            // TODO Resolve race window between previous polling and request_notification
            ret = m_p_cq_mgr_tx->request_notification();
"
also need to handle I think

Reply:
I remove all the request_notification with polling mechanism for DOCA. These are just leftovers for dpcp path that will be removed with DPCP. I don't see a sense in all this. If there are no wqes then the sq is too small or it just always full in which case 99% this attempt will always fail. The same applies to the mem_buf_tx_get.
That's why i don't see a reason to fix the old path.